### PR TITLE
update common and fix race test condition

### PR DESCRIPTION
**Pull request by khmic5**
# Contributing to the Azure SDK

Please see our [CONTRIBUTING.md](https://github.com/Azure/azure-sdk-for-net/blob/main/CONTRIBUTING.md) if you are not familiar with contributing to this repository or have questions.

For specific information about pull request etiquette and best practices, see [this section](https://github.com/Azure/azure-sdk-for-net/blob/main/CONTRIBUTING.md#pull-request-etiquette-and-best-practices).

### CODE REVIEW

File: sdk/maps/Azure.Maps.Search/tests/FuzzySearchTests.cs [hunk 1]
```diff
@@ -72,6 +72,8 @@ public async Task CanPollFuzzySearchBatch()
                 new FuzzySearchQuery("pizza", new FuzzySearchOptions { Coordinates = new GeoPosition(121.56, 25.04) }),
             });
 
+            // delay 400 ms for the task to complete
+            await Task.Delay(400);
```
Comment (weshaggard) [inline sdk/maps/Azure.Maps.Search/tests/FuzzySearchTests.cs:76]: Is this delay required? What do we expect customers to do here? It seems like our wait should actually wait for completion as the name implies.

Comment (khmic5) [reply sdk/maps/Azure.Maps.Search/tests/FuzzySearchTests.cs:76]: This delay in the test case prevents the race condition error from happening so there is enough time to retrieve the batch result

Comment (khmic5) [reply sdk/maps/Azure.Maps.Search/tests/FuzzySearchTests.cs:76]: https://github.com/Azure/azure-sdk-for-net/issues/31019

Comment (khmic5) [reply sdk/maps/Azure.Maps.Search/tests/FuzzySearchTests.cs:76]: I have opened an issue to track what appears to be a server side bug

Comment (jsquire) [reply sdk/maps/Azure.Maps.Search/tests/FuzzySearchTests.cs:76]: I had the same concern.  We chatted offline; it appears this is a service bug where the operation request returns a 202 but the subsequent status poll returns a 404.   The Maps team is going to investigate.   For now, we'll keep the delay using #31019 to track the debt of revisiting once the service bug is fixed.

**Review by jsquire — APPROVED** [review]

**Pull request merged**